File: src/codes/Users_Manual/geometry_4.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Link;
~using AUTD3Sharp.Utils;
~using var autd = Controller.Builder([new AUTD3(Vector3.Zero)]).Open(Nop.Builder());
~{
  var dev = autd.Geometry[0];
  ~}
foreach (var dev in autd.Geometry)
{
  // do something
}

File: src/codes/Users_Manual/geometry_4.cpp
//~#include<chrono>
//~#include<autd3.hpp>
//~#include<autd3/link/nop.hpp>
//~int main() {
//~auto autd =
//~autd3::ControllerBuilder().add_device(autd3::AUTD3(autd3::Vector3::Zero())).open(autd3::link::Nop::builder());
//~{
auto dev = autd.geometry()[0];
//~}
//~{
for (auto& dev : autd.geometry()) {
  // do something
}
//~}
//~return 0; }